portal/config/eproms/CommunicationRequest.json

```diff
@@ -377,7 +377,7 @@
         }
       ],
       "lr_uuid": "b71dfdf6-bb19-3fff-84bd-4cdb30050499",
-      "notify_post_qb_start": "{\"days\": 0}",
+      "notify_post_qb_start": "{\"month\": 1, \"days\": 0}",
```

Comment:
The internals of RelativeDelta are a pain - to get around month versus months comparison issues, we enforce all such nouns to be in their plural form - that's why the tests are failing